js-api-doc/value/index.d.ts

```diff
@@ -17,40 +17,144 @@ export {SassMap} from './map';
 
+  /**
+   * Converts `sassIndex` into a JavaScript-style index into the list returned
+   * by [[asList]].
+   *
+   * Sass indexes are one-based, while JavaScript indexes are zero-based. Sass
+   * indexes may also be negative in order to index from the end of the list.
+   *
+   * @param sassIndex - The Sass-style index into this as a list.
+   * @param name - The name of the function argument `sassIndex` came from
+   * (without the `$`) if it came from an argument. Used for error reporting.
+   * @throws `Error` If `sassIndex` isn't a number, if that number isn't an
+   * integer, or if that integer isn't a valid index for [[asList]].
+   */
   sassIndexToListIndex(sassIndex: Value, name?: string): number;
 
+  /**
+   * Throws if `this` isn't a [[SassBoolean]].
+   *
+   * **Heads up!** Functions should generally use [[isTruthy]] rather than
+   * requiring a literal boolean.
+   *
+   * @param name - The name of the function argument `this` came from (without
+   * the `$`) if it came from an argument. Used for error reporting.
+   */
   assertBoolean(name?: string): SassBoolean;
 
+  /**
+   * Throws if `this` isn't a [[SassColor]].
+   *
+   * @param name - The name of the function argument `this` came from (without
+   * the `$`) if it came from an argument. Used for error reporting.
+   */
   assertColor(name?: string): SassColor;
 
+  /**
+   * Throws if `this` isn't a [[SassFunction]].
+   *
+   * @param name - The name of the function argument `this` came from (without
+   * the `$`) if it came from an argument. Used for error reporting.
+   */
   assertFunction(name?: string): SassFunction;
 
+  /**
+   * Throws if `this` isn't a [[SassMap]].
+   *
+   * @param name - The name of the function argument `this` came from (without
+   * the `$`) if it came from an argument. Used for error reporting.
+   */
   assertMap(name?: string): SassMap;
 
+  /**
+   * Throws if `this` isn't a [[SassNumber]].
+   *
+   * @param name - The name of the function argument `this` came from (without
+   * the `$`) if it came from an argument. Used for error reporting.
+   */
   assertNumber(name?: string): SassNumber;
 
+  /**
+   * Throws if `this` isn't a [[SassString]].
+   *
+   * @param name - The name of the function argument `this` came from (without
+   * the `$`) if it came from an argument. Used for error reporting.
+   */
   assertString(name?: string): SassString;
 
+  /**
+   * Returns the map contents of `this` if it's a map (including empty lists, which
+   * count as empty maps) or `null` if it's not.
+   *
+   * @returns An immutable [[OrderedMap]] from the [`immutable`
+   * package](https://immutable-js.com/), or `null`.
+   */
   tryMap(): OrderedMap<Value, Value> | null;
 
+  /** Returns whether `this` represents the same value as `other`. */
   equals(other: Value): boolean;
 
+  /** Returns a hash code that can be used to store `this` in a hash map. */
   hashCode(): number;
 
+  /** @hidden */
```

Comment:
It doesn't need to, but it's kind of unsightly in the generated documentation and doesn't really provide any additional information since `Object.toString()` already exists.